topics/imaging/tutorials/tutorial-CP/tutorial.md

```diff
@@ -0,0 +1,633 @@
+>    - *"Use objects or an image as a mask?"*: `Objects`
+>        - *"Enter the name objects to mask the input image"*: `Nuclei`
+>    - *"Invert the mask?"*: `No`
+>
+>
+{: .hands_on}
+
+
+Now that we have all the holes in one mask, we can segment the nucleoli as individual objects in the same way as we did with the nuclei. All the nucleoli can be then combined into one single image.
+
+> ### {% icon hands_on %} Hands-on: Segment nucleoli as individual objects
+>
+> 1. **IdentifyPrimaryObjects** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **MaskImage** {% icon tool %})
+>    - *"Use advanced settings?"*: `Yes, use advanced settings`
+>        - *"Enter the name of the input image (from NamesAndTypes)"*: `MaskDNAdarkholes`
+>        - *"Enter the name of the primary objects to be identified"*: `Nucleoli`
+>        - *"Typical minimum diameter of objects, in pixel units (Min)"*: `2`
+>        - *"Typical maximum diameter of objects, in pixel units (Max)"*: `15`
+>        - *"Discard objects touching the border of the image?"*: `Yes`
+>        - *"Threshold strategy"*: `Global`
+>            - *"Thresholding method"*: `Otsu`
+>                - *"Two-class or three-class thresholding?"*: `Two classes`
+>                - *"Threshold correction factor"*: `0.9`
+>        - *"Method to distinguish clumped objects"*: `Shape`
+>            - *"Method to draw dividing lines between clumped objects"*: `Shape`
+>                - *"Automatically calculate size of smoothing filter for declumping?"*: `Yes`
+>                - *"Automatically calculate minimum allowed distance between local maxima?"*: `Yes`
+>        - *"Handling of objects if excessive number of objects identified"*: `Continue`
+>
+>
+> 2. **ConvertObjectsToImage** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **IdentifyPrimaryObjects** {% icon tool %})
+>    - *"Enter the name of the input objects you want to convert to an image"*: `Nucleoli`
+>    - *"Enter the name of the resulting image"*: `MaskNucleoli`
+>    - *"Select the color format"*: `Binary (black & white)`
+>
+>
+{: .hands_on}
+
+
+## Combine segmentation masks
+
+We have now one segmentation mask per image with all the nuclei detected, `MaskNuclei`, and another one for the nucleoli, `MaskNucleoli`. These are binary masks in which the background is black and the objects detected are white. We would like to check whether both segmentation steps went well. That could be achieved by combining both (using different colors) into one image. Here we are converting the nucleus mask to blue and the nucleoli to magenta. The outcome will look like:
+
+![Combined mask for nuclei and nucleoli](../../images/tutorial-CP/img_combined_masks.png "Nuclei and nucleoli masks combined in which the nuclei are in blue and nucleoli in magenta.")
```

Comment:
Here too it would be nice to be able to make this image.

Reply:
same as above... it's not possible so I've clarified it in a sentence